.circleci/config.yml

```diff
@@ -226,6 +226,7 @@ workflows:
                 - thumbnail
                 - pkgselect
                 - shared
+                - pkgevents
```

Comment:
pkgevents is too specific? try events, stackevents, etc?

Reply:
Still feels too narrow; eventbridge or events is recommended to accommodate folder, object, etc. events